src/components/navigation/Navigation.tsx

```diff
@@ -14,24 +14,34 @@ function Navigation() {
   const [navState, setNavState] = useState(NavigationState.expanded);
   const navigation = NavigationService.getNavigation();
 
+  const navigationClasses = cx('Navigation', {
+    expanded: navState === NavigationState.expanded,
+    collapsed: navState === NavigationState.collapsed
+  });
+
+  const navIconClasses = cx('nav-icon', { collapsed: navState === NavigationState.collapsed });
+
   return (
     // TODO: is there a better way to do conditional classes. What if there are multiple conditionals?
-    <div className={navState === NavigationState.expanded ? 'Navigation expanded' : 'Navigation collapsed'}>
+    // Not natively but there is a popular library called classnames that does this well
+    <div className={navigationClasses}>
       <div className="title-and-icon p-2">
         <FontAwesomeIcon
-          className={navState === NavigationState.expanded ? 'nav-icon' : 'nav-icon collapsed'}
+          className={navIconClasses}
           icon={faCoffee}
           size="2x"
           onClick={() => setNavState(navState === NavigationState.collapsed ? NavigationState.expanded : NavigationState.collapsed)}
         />
         <h2 className={navState === NavigationState.expanded ? 'title' : 'title d-none'}>{navigation.title}</h2>
         {/* <h2 className="title d">{navState === NavigationState.expanded ? navigation.title : ''}</h2> */}
       </div>
-      {navigation.items.map((navItem: NavigationItemModel) => (
```

Comment:
Not React related but you don't need to specify the TS Type here as TS can identify it by itself 😃